### PR TITLE
Fix comments to match usage

### PR DESCRIPTION
**Pull request by Rholais**
Fix comments of `sparse_softmax_cross_entropy_with_logits` to match usage.

### CODE REVIEW

**Review by protoget — APPROVED** [review]

**Comment by tensorflowbutler** [comment]
Nagging Assignee @protoget: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @protoget: It has been 29 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.
